update-npm.rb

```diff
@@ -0,0 +1,102 @@
+require "dependabot/omnibus"
+
+package_manager = "npm_and_yarn"
+repo = "YOUR_ORG/YOUR_PROJECT/_git/YOUR_REPO"
+
+credentials = [{
+  "type" => "git_source",
+  "host" => "dev.azure.com",
+  "username" => "",
+  "password" => ENV["SYSTEM_ACCESSTOKEN"]
+},{
+  "type" => "npm_registry",
+  "registry" => "YOUR_ORG.pkgs.visualstudio.com/_packaging/YOUR_FEED/npm/registry/",
+  "token" => "YOUR_FEED:" + ENV["SYSTEM_ACCESSTOKEN"]
+}]
```

Comment:
You might also want to add `{
  "type" => "git_source",
  "host" => "github.com",
  "username" => "x-access-token",
  "password" => "<token>"
}` here.